tests/Core/Controller/OCSControllerTest.php

```diff
@@ -88,6 +88,10 @@ public function testGetCapabilities() {
 		$this->userSession->expects($this->once())
 			->method('isLoggedIn')
 			->willReturn(true);
+		$this->request
+			->method('getServerHost')
+			->willReturn('awesomehost.io');
```

Comment:
```suggestion
			->willReturn('example.tld');
```
or a nextcloud domain